Makefile

```diff
@@ -217,7 +217,7 @@ $(MAGFRM):
 	$(MAKE) -C tools/dasm
 
 $(TV11):
-	$(MAKE) -C tools/tv11
+	$(MAKE) -C tools/tv11 CFLAGS=-O3
```

Comment:
@aap, I had to do this to work around an old GCC which doesn't understand -fno-diagnostics-color and -fno-diagnostics-show-caret.